arch/risc-v/src/mpfs/mpfs_opensbi.c

```diff
@@ -603,6 +580,11 @@ void __attribute__((noreturn)) mpfs_opensbi_setup(void)
 {
   uint32_t hartid = current_hartid();
 
+  for (i = 0; i <  sizeof(mpfs_hart_index2id / sizeof(u32); i++)
```

Comment:
should be int?

Reply:
The / sizeof(u32) should be replaced by / sizeof(mpfs_hart_index2id[0]) so if the type / size changes (it never will though..) this loop will still work

Reply:
What, is someone saying that I should try to compile the code before submitting a PR? Thanks for review, and apologies :smile: